linera-rpc/proto/rpc.proto

```diff
@@ -53,13 +53,21 @@ service ValidatorNode {
   rpc GetVersionInfo(google.protobuf.Empty) returns (VersionInfo);
 }
 
+// Information about the Linera crate version the validator is running
+message CrateVersion {
+    uint64 major = 1;
```

Comment:
Would `u32` be more appropriate for version components? (also thinking of javascript)

Reply:
`semver` uses `u64`, but yes, maybe it's better to use `u32`.  (honestly if a version number gets past `u8` I'd be surprised!)